src/main/java/redis/clients/jedis/GeoRadiusResponse.java

```diff
@@ -34,4 +35,12 @@ public double getDistance() {
   public GeoCoordinate getCoordinate() {
     return coordinate;
   }
+
+  public long getScore() {
```

Comment:
@yangbodong22011 Redis defines it as `raw geohash-encoded sorted set score`. WDYT about naming the variable as `RawScore` instead of just `Score`? Just to be clearer (for now) and safer (for future).

Reply:
@sazzad16 Good suggestion, I have submitted the modification code.